Sources/Playbook/Design Elements/Shadows/PBShadow.swift

```diff
@@ -22,16 +22,16 @@ extension View {
 }
 
 public enum Shadow: String, CaseIterable {
-  case deep, deeper, deepest, none
-
+  case deep, deeper, deepest, top, none
```

Comment:
Do we have. a new token for shadow?

Reply:
Nope, I made that when I was just trying things out. I removed it and added a .shadow() method to the component instead. Not sure why its still showing as being added

Reply:
It looks like I removed it everywhere except the enum. It's gone now. I committed again